projectsCMSFunctions.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+//var_dump($_POST);
+
+function setUpPDO () {
+
+    $db = new PDO('mysql:host=127.0.0.1;dbname=portfolio-kevin', 'root');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+}
+
+function selectProjectImage () {
```

Comment:
this function isn't doing anything as it is a select statement that doesnt have a return statement